wpilibj/src/athena/java/edu/wpi/first/wpilibj/Preferences.java

```diff
@@ -81,15 +82,11 @@ private Preferences() {
   }
 
   /**
-   * Gets the vector of keys.
-   * @return a vector of the keys
+   * Gets a List of keys.
+   * @return a List of the keys
    */
-  public Vector getKeys() {
-    Vector<String> keys = new Vector<String>();
-    for (String key : m_table.getKeys()) {
-      keys.add(key);
-    }
-    return keys;
+  public List<String> getKeys() {
```

Comment:
Pretty sure I wanted to change this at one point, but we decided to not do a breaking change.